internal/config/config.go

```diff
@@ -162,24 +203,6 @@ func (c *AuthConfig) User(hostname string) (string, error) {
 	return c.cfg.Get([]string{hosts, hostname, "user"})
 }
 
-// GitProtocol will retrieve the git protocol for the logged in user at the given hostname.
-// If none is set it will return the default value.
-func (c *AuthConfig) GitProtocol(hostname string) string {
```

Comment:
This is now moved off of `AuthConfig` and onto `Config`.

Reply:
Is the problem with this old code that it didn't return a final default value if an error happens?

_I'm having trouble tracing this back to a specific issue to make sure I understand the problem being fixed_ 🙇